util/execute.go

```diff
@@ -25,3 +25,12 @@ func Execute(args ...string) {
 		log.Fatalf("%s", output)
 	}
 }
+
+// ExecuteWithOutput runs the given strings as a command and returns the output.
+func ExecuteWithOutput(args ...string) string {
```

Comment:
Would love a better spot for this than `package util`. "Avoid meaningless package names. Packages named util, common, or misc provide clients with no sense of what the package contains." --https://blog.golang.org/package-names

If these are only used in main.go, we could just move them there.

Reply:
Ah yes, it is only used in `cmd/release` so I will move it there. I copied a lot of this stuff from another repo that had more utilities depending on `execute.go`. Didn't think twice about it when I moved it over.